admin/assets/javascripts/global-scripts.js

```diff
@@ -475,6 +475,11 @@ $(function () {
       groupFilters.tableRowVisibility(groupIds)
       stickyBanner.outputCheckedCheckboxes(inputStatus.countCheckedCheckboxes())
       stickyBanner.stickyBannerPositioning()
+      if (pupilsNotTakingCheck.isCheckboxChecked()) {
+        stickyBanner.toggle(true)
+      } else {
+        stickyBanner.toggle(false)
+      }
```

Comment:
Just a minor improvement that can be achieved to reduce the code:
`const isCheckboxChecked = !!pupilsNotTakingCheck.isCheckboxChecked() //handles null or undefined as well`
`stickyBanner.toggle(isCheckboxChecked)`

Reply:
that 'improvement' is very subjective

Reply:
i prefer the original code